lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -10,12 +10,15 @@
  * - tenantId               - Full tenant or organizational id that this account belongs to
  * - username               - preferred_username claim of the id_token that represents this account
  * - name                   - Full name for the account, including given name and family name
+ * - idTokenClaims          - Object contains claims from ID token
+ * - localAccountId         - The user's account ID
  */
 export type AccountInfo = {
     homeAccountId: string;
     environment: string;
     tenantId: string;
     username: string;
     name?: string;
+    idTokenClaims?: object;
```

Comment:
Can we make `localAccountId` mandated with this PR instead of a new PR?

Reply:
We can, although its a bit unrelated.

Reply:
Never mind. #2559 is addressing it.